packages/start-client-core/src/serializer.ts

```diff
@@ -1,17 +1,12 @@
-import { isPlainObject } from './utils'
+import { isPlainObject } from '@tanstack/router-core'
 
-export interface TsrSerializer {
+export interface StartSerializer {
```

Comment:
Why does the Serializer get moved to Start? why not keep it in `router-core`.

Wouldn't this same serializer benefit normal `basic-file-based-ssr-streaming` as well?

Reply:
this PR only uses seroval for SSR
for server functions, we still use the old serializer. in a followup PR, we'll move server functions over to seroval as well